stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -291,25 +296,39 @@ public class ChatClient internal constructor(
 
             override fun enqueue(callback: Call.Callback<ConnectionData>) {
```

Comment:
Can't we just use `CoroutineCall` here?

Reply:
Hey @adasiewiczr 

Thanks for your feedback.
I didn't notice we have something like this 🙈

Yes we def can use `CoroutineCall` here 👍